WORKSPACE

```diff
@@ -150,7 +151,13 @@ maven_jar(
 
 maven_jar(
   name = "org_apache_httpcomponents_http_client",
-  artifact = "org.apache.httpcomponents:httpclient:4.5.2",
+  artifact = "org.apache.httpcomponents:httpclient:" + http_client_version,
+)
+
+http_jar(
```

Comment:
Apache Http Client test dependency requires `classifier` flag with `tests` value. To achieve this, `http_jar` support has been used.

Reply:
@erenavsarogullari - is it not possible to use maven jar?

Reply:
@kramasamy -  Thanks for review this. Currently, Bazel looks having an open PR to support `classifier` functionality. I have tried with `maven-jar` as well. Also, classifier is supported by Maven and Gradle. 
https://github.com/johnynek/bazel-deps/issues/69